freddytoolapi/Freddy.UnitTests/API/Controllers/CustomerController/DeleteCustomer.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Freddy.Application.Commands.Customers;
+using Freddy.Application.Core.Commands;
+using Freddy.Application.Core.Queries;
+using Moq;
+using Xunit;
+
+namespace Freddy.Application.UnitTests.API.Controllers
+{
+    public static partial class CustomerControllerTests
+    {
+        public class DeleteCustomer : IClassFixture<EmptyWebApplicationFactory<TestStartup>>
```

Comment:
Myslim ze mozes pouzit CommonWebApplicationBuilder

Reply:
+1 you are right, nevsimol som si